.github/workflows/version.yml

```diff
@@ -0,0 +1,49 @@
+name: Manually Triggered Tag and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to tag and release'
+        required: true
+
+jobs:
+  check-and-create-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Check if tag exists
+        id: check-tag
+        run: |
+          if git tag | grep -q "${{ github.event.inputs.version }}"; then
+            echo "Tag exists"
+            echo "exists=1" >> "$GITHUB_OUTPUT"
+          else
+            echo "Tag does not exist"
+            echo "exists=0" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Create and push tag
```

Comment:
Curious why you decided not to split this into two jobs like spice.js and spicepy?

Reply:
Or why you decided to split spice.js and spicepy into two jobs instead of combining them like this.

Reply:
Discussed in chat - we will split this into two jobs to match the other SDKs